aiidalab/app.py

```diff
@@ -8,14 +8,14 @@
 import sys
 import tarfile
 import tempfile
+import subprocess
```

Comment:
Looks like this is not enforced yet, but imports should be alphabetically.

Reply:
done